pagopa/io-app
# feat: [IA-362,IA-383] Accessibility: fixed organization's fiscal code copy button

## Short description
Fixed organization's fiscal code copy button in services by changing its accessibility label.

## List of changes proposed in this pull request
- [locales/en/index.yml,locales/it/index.yml]: added a label to speech the right text when the organization's fiscal code copy button is pressed.
- ts/components/services/ServiceMetadata/index.tsx: implemented the new accessibility label.

## How to test
Run app, enable VoiceOver (or TalkBack) and verify if the new label is read when in services you try to copy the organization's fiscal code
